apps/web-portal/.env.defaults

```diff
@@ -0,0 +1 @@
+SERVER_PORT=7777
```

Comment:
`ZORKO_API_HTTP_PORT`

let's prefix all our env variables as `ZORKO_` to not intersect with other systems

Reply:
done